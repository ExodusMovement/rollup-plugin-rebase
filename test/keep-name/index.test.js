import { bundle, clean, list, read } from "../util"

const root = __dirname

test("Keep Name", async () => {
  await bundle(root, "index.js", "output/index.js", { keepName: true })

  expect(await list(root, "output")).toMatchSnapshot()
  expect(await read(root, "output/index.js")).toMatchSnapshot()
  expect(await read(root, "output/index~gToHHcDx.css")).toMatchSnapshot()

  await clean(root, "output")
})
