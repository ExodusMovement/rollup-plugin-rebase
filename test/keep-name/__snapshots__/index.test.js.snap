// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Keep Name 1`] = `
"content~HDWIGpiP.md
image~SmVOQyTt.png
index.js
index~rfFBfmSa.css
index~rfFBfmSa.css.map"
`;

exports[`Keep Name 2`] = `
"import content from './content~HDWIGpiP.md';
import css from './index~rfFBfmSa.css';

function index() {
  return content + css
}

export { index as default };
"
`;

exports[`Keep Name 3`] = `
"body {
  font-size: 20px;
  background: url(./image~SmVOQyTt.png) top left repeat;
}

/*# sourceMappingURL=index~rfFBfmSa.css.map */"
`;
